app/moves/middleware.js

```diff
@@ -90,4 +94,36 @@ module.exports = {
       next(error)
     }
   },
+  setMovesByDateAllLocations: async (req, res, next) => {
+    const { moveDate } = res.locals
+
+    if (!moveDate) {
+      return next()
+    }
+
+    try {
+      const userLocations = get(req.session, 'user.locations', []).map(
+        loc => loc.id
```

Comment:
Maybe use more verbose variable names like we've been using so far: `location => location.id`. I find more verbose names help when returning to code after long periods of time.